app/models/rent.rb

```diff
@@ -0,0 +1,6 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+
+  validates :user, :book, :start_date, :end_date, presence: true
```

Comment:
I let you [this doc](https://guides.rubyonrails.org/v3.2.13/active_record_validations_callbacks.html#presence) to try to correct this line ;)